src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -33,13 +46,30 @@ public String getColor() {
         return color;
     }
 
-    public void setName(String name) {
+    public Sections getSections() { return sections; }
+
+    public void addSection(Station upStation, Station downStation, Distance distance) {
+        Section section = Section.builder()
+                .line(this)
+                .upStation(upStation)
+                .downStation(downStation)
+                .distance(distance.getValue())
+                .build();
+
+        sections.validateAddSection(upStation, downStation);
+        sections.add(section);
```

Comment:
validateAddSection 메서드에서 예기치못한 예외가 발생할 수 있는데, 이 코드만 봐서는 전혀 알 수 없네요 ㅠ

그리고 이 구간 정보를 추가할 수 있는지 없는지에 대한 검사도 노선보다는 구간에서 해주는게 적절한 책임이 아닐까 싶어요 ㅎ 

이런식으로 바꿀 수는 없을까요?
```suggestion
        sections.add(section);
```

구간 추가가 가능한지 여부는 sections 자체가 검사하는거죠 ㅎ